web3j-ext/web3j-ext/web3jext-generate.sh

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+#!/usr/bin/env bash -e
```

Comment:
`-e` option in the shebang works if you run like `$ ./script.sh`
But the shebang is ignored if you run like `$ sh script.sh` or `$ bash script.sh`.

Instead, add a separate `set -e` line please.